### ISSUE TITLE
Running a default application dies with missing method

### ISSUE THREAD

**Issue by jjatria** [perlpunk/App-AppSpec-p5#2]
When generating a new app, the generated module has the default command name `mycommand`, and lacks an `execute` sub, which seems to be called by default when no arguments are provided.

This makes it so that running this series of commands

```perl
appspec new --name myapp --class My::App
perl -IMy-App/lib My-App/bin/myapp
```

results in the following error

```
Can't locate object method "execute" via package "My::App" at /path/to/lib/App/Spec/Run.pm line 93.
```

**Comment by perlpunk** [comment]
thanks, I'll try to look at it soon; very busy during the next 2 weeks

**Comment by perlpunk** [comment]
Thanks, I've merged #3
